docs/community/topics/i18n.rst

```diff
@@ -65,13 +65,25 @@ These steps cover how to add or change text that has been marked as translateabl
 
       pybabel extract . -F babel.cfg -o src/pydata_sphinx_theme/locale/sphinx.pot -k '_ __ l_ lazy_gettext'
 
+   **To run this in ``.nox``**: ``nox -s translate -- extract``.
+
 #. Update the message catalogs (``PO`` files) with `the PyBabel update command <https://babel.pocoo.org/en/latest/cmdline.html#update>`__:
 
    .. code-block:: bash
 
       pybabel update -i src/pydata_sphinx_theme/locale/sphinx.pot -d src/pydata_sphinx_theme/locale -D sphinx
 
+   **To run this in ``.nox``**: ``nox -s translate -- update``.
+
+
 This will update these files with new information about the position and text of the language you have modified.
+If you change non-translatable text, you can run pybabel and it will update the line numbers.
+However, this step is optional - the line numbers are to inform the human translator, not to perform the translation.
+
+If you change translatable text, pybabel will try to fuzzy match the new text with an existing translation.
+If so it will add a comment like `#, fuzzy`.
+Otherwise, the new text will simply be untranslated in the file.
```

Comment:
```suggestion
If you change translatable strings, the `extract` command will extract the new or updated strings to the POT file, and the `update` command will try to fuzzy match the new or updated strings with existing translations in the PO files.
If there is a fuzzy match, a comment like `#, fuzzy` is added before the matched entry.
Otherwise, a new entry is added and needs to be translated.
```